.github/workflows/frontend.yml

```diff
@@ -30,6 +30,18 @@ jobs:
           path: |
             **/node_modules
           key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+      - name: build cache
+        if: |
+          steps.node-modules-cache.outputs.cache-hit == 'true' &&
+          github.ref != 'refs/heads/main'
+        id: build-cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            **/dist
+            **/tsconfig.tsbuildinfo
+            !**/node_modules
```

Comment:
just curious is this actually necessary? are there `node_modules` in dist? can't hurt either way i guess

Reply:
This is to avoid caching dist directories under the node_modules directory